src/patterns/FavoritesCollectionTile/FavoritesCollectionTile.tsx

```diff
@@ -0,0 +1,249 @@
+import * as Popover from '@radix-ui/react-popover';
+import classnames from 'classnames';
+import { ComponentProps, ElementType, forwardRef, memo, useRef } from 'react';
+import { Icon } from '../../components/Icon';
+import Link, { LinkProps } from '../../components/Link/Link';
+import { SeldonImage } from '../../components/SeldonImage';
+import { Text, TextVariants } from '../../components/Text';
+import { getCommonProps } from '../../utils';
+
+export interface FavoritesCollectionTileProps extends ComponentProps<'div'> {
+  /**
+   * Unique identifier for the list
+   */
+  id: string;
+  /**
+   * Lots display text en/zh
+   */
+  lotText?: string;
+  /**
+   * Lots display text en/zh
+   */
+  lotsText?: string;
+  /**
+   * Lot empty Favorites text en/zh
+   */
+  emptyFavoritesText?: string;
+  /**
+   * Lot blank list text en/zh
+   */
+  blankListText?: string;
+  /**
+   * Lot empty lists text en/zh
+   */
+  emptyListsText?: string;
+  /**
+   * Lot edit list text en/zh
+   */
+  editListText?: string;
+  /**
+   * Lot delete list text en/zh
+   */
+  deleteListText?: string;
+  /**
+   * List data count
+   */
+  count: number;
+  /**
+   * List data name
+   */
+  name: string;
+  /**
+   * Image URL for the list
+   */
+  listImageUrl: string;
+  /**
+   * Whether this card is in favorites view
+   */
+  isFavorites?: boolean;
+  /**
+   * Whether this card is in lists view
+   */
+  isLists?: boolean;
+  /**
+   * Element to render within the navigation item, renders <Link> by default
+   */
+  element?: ElementType<LinkProps>;
+  /**
+   * imageLink
+   */
+  imageLink?: string;
+  /**
+   * Callback function for editList menu item click
+   */
+  onEdit?: (id: string, event: React.MouseEvent<HTMLButtonElement>) => void;
+  /**
+   * Callback function for deleteList menu item click
+   */
+  onDelete?: (id: string, event: React.MouseEvent<HTMLButtonElement>) => void;
+  /**
+   * Function to modify strings for lot string translation
+   */
+  formatlotStr?: (count: number) => string;
+}
+
+/**
+ * ## Overview
+ *
+ * Overview of this widget
+ *
+ * [Figma Link](https://www.figma.com/design/rIefa3bRPyZbZmtyV9PSQv/My-Account?node-id=61-14355&m=dev)
+ *
+ * [Storybook Link](https://phillips-seldon.netlify.app/?path=/docs/patterns-FavoritesCollectionTile--overview)
+ */
+const FavoritesCollectionTile = memo(
+  forwardRef<HTMLDivElement, FavoritesCollectionTileProps>(
+    (
+      {
+        id,
+        className,
+        count,
+        name,
+        listImageUrl,
+        isLists = false,
```

Comment:
do we need both a `isLists` AND `isFavorites` prop or isn't is one or the other? could we make this a `variant` with an enum of `favorites` and `lists`